spec/features/event/show_event_spec.rb

```diff
@@ -203,6 +203,18 @@
       end
     end
 
+    context "for events whose startdate has passed" do
+      before(:each) do
+        @ongoing_event = FactoryBot.create :event, startdate: Date.yesterday
+        sign_in @user
+        visit event_path(@ongoing_event)
+      end
+
+      it "should not display a delete button" do
+        expect(page).not_to have_css('a', :text => I18n.t('helpers.links.destroy'))
+      end
+    end
+
```

Comment:
If we're working with cucumber now this test isn't needed anymore. Cucumber replaces all our feature tests.